lib/matplotlib/quiver.py

```diff
@@ -239,6 +174,53 @@ def __init__(self, Q, X, Y, U, label,
                  *, angle=0, coordinates='axes', color=None, labelsep=0.1,
                  labelpos='N', labelcolor=None, fontproperties=None,
                  **kw):
+        """
+        Add a key to a quiver plot.
+
+        The positioning of the key depends on *X*, *Y*, *coordinates*, and
+        *labelpos*.  If *labelpos* is 'N' or 'S', *X*, *Y* give the position of
+        the middle of the key arrow.  If *labelpos* is 'E', *X*, *Y* positions
+        the head, and if *labelpos* is 'W', *X*, *Y* positions the tail; in
+        either of these two cases, *X*, *Y* is somewhere in the middle of the
+        arrow+label key object.
+
+        Parameters
+        ----------
+        Q : `Quiver`
+            A `Quiver` object as returned by a call to `~Axes.quiver()`.
```

Comment:
```suggestion
            A `.Quiver` object as returned by a call to `~.Axes.quiver()`.
```
`~Axes.quiver()` does only link for `Axes.quiverkey` not for `pyplot.quiverkey()`. The leading dot enables non-complete matches also in to other modules.

Not sure if the dot fixes `.Quiver` as well, but possibly yes since the actual link is from axes/pyplot to quiver_api.